tests/beaker_tests/cisco_vxlan_vtep_vni/test_vxlan_vtep_vni.rb

```diff
@@ -337,10 +340,10 @@ def version_unsupported_properties(_tests, _id)
   # test_harness_run(tests, :suppress_arp_true)
   # test_harness_run(tests, :suppress_arp_false)
 
-  test_harness_run(tests, :suppress_uuc_true)
-  test_harness_run(tests, :suppress_uuc_false)
-  test_harness_run(tests, :multisite_ingress_replication_true)
-  test_harness_run(tests, :multisite_ingress_replication_false)
+  test_harness_run(tests, :suppress_uuc_true, TestVxLanVtepVni.method(:test_harness_dependencies))
+  test_harness_run(tests, :suppress_uuc_false, TestVxLanVtepVni.method(:test_harness_dependencies))
+  test_harness_run(tests, :multisite_ingress_replication_true, TestVxLanVtepVni.method(:test_harness_dependencies))
+  test_harness_run(tests, :multisite_ingress_replication_false, TestVxLanVtepVni.method(:test_harness_dependencies))
```

Comment:
Looking at this, I more and more don't like it. There is too much duplication with the repeated `method()` calls, and it doesn't look like good ruby.

Please rework this instead to pass in the class itself, and have the `test_harness_run` method call the right thing:

```ruby
test_harness_run(tests, :multisite_ingress_replication_false, TestVxLanVtepVni)
```

and

```ruby
def test_harness_run(tests, id, harness_class=nil, skip_idempotence_check=false)
  # ...
  harness_class.test_dependencies(tests, id) if harness_class
```